Source/DafnyCore/DafnyOptions.cs

```diff
@@ -1444,15 +1444,16 @@ verification outcome.
     option is useful in a diamond dependency situation, to prevent code
     from the bottom dependency from being generated more than once.
 
-/testContracts:<AllExterns|ExternsInTests>
+/testContracts:<Externs|TestedExterns>
     Enable run-time testing of compiled function or method contracts in
```

Comment:
I just realize one important thing. You are not testing "compiled functions", you are testing "compilable (function or method) contracts". Could you please rephrase it like `extern function or method contracts that can be compiled, at the call site`? I think it's important to mention the call site, because, until this comment, I thought you were wrapping the functions themselves, but you are actually rewriting every call to that function or method.

Reply:
Ah, yeah, my original intent of the word "compiled" was just to say that this only applied to functions and methods that are being compiled. I think your use of the word is better, though! So I'll change the text accordingly.